code/zFulpstationCode/fulp_overwrite_vars.dm

```diff
@@ -281,7 +322,118 @@
 		/obj/item/stack/tile/plasteel/cyborg,
 		/obj/item/borg/apparatus/circuit,
 		/obj/item/stack/cable_coil/cyborg)
-
+		
+/obj/item/robot_module/peacekeeper
+	basic_modules = list(
+		/obj/item/laser_pointer, //replaced flash
```

Comment:
you even removed them from PEACEKEEPER CYBORGS?

Reply:
Might be worth leaving in, nobody place peacekeeper as it is anyway